### PR TITLE
refactor: use stacks query pkg

### LINKED ISSUE #5070: Fix formatting for transactions in submitted state in activity
Screenshot 2024-03-13 at 11 32 23

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out Leather build caa3382 — [Extension build](https://github.com/leather-wallet/extension/actions/runs/9469964098), [Test report](https://leather-wallet.github.io/playwright-reports/feat-stacks-query-pkg), [Storybook](https://feat-stacks-query-pkg--65982789c7e2278518f189e7.chromatic.com), [Chromatic](https://www.chromatic.com/library?appId=65982789c7e2278518f189e7&branch=feat-stacks-query-pkg)

This PR removes the stacks queries from the extension and installs them from the mono repo. Once these are removed, and the code exists in one place, I will refactor the queries and some code might be brought back.

## Summary by CodeRabbit

- **New Features**
  - Introduced a new function to filter assets based on positive balances.
  - Updated the `Collectibles` component to handle account addresses more efficiently.
  
- **Bug Fixes**
  - Improved handling of account data refresh and transaction updates.

- **Refactor**
  - Consolidated and reorganized various imports to improve code maintainability.
  - Updated multiple hooks to use new query sources for better performance and consistency.

- **Chores**
  - Updated package dependencies to their latest versions for enhanced security and functionality.

### CODE REVIEW

File: src/app/pages/send/send-crypto-asset-form/form/stacks/use-stacks-common-send-form.tsx [hunk 1]
```diff
@@ -27,8 +27,8 @@ export function useStacksCommonSendForm({
   availableTokenBalance,
 }: UseStacksCommonSendFormArgs) {
   const routeState = useSendFormRouteState();
-  const { data: nextNonce } = useNextNonce();
-  const currentAccountStxAddress = useCurrentStacksAccountAddress();
+  const stxAddress = useCurrentStacksAccountAddress();
```
Comment (pete-watters) [inline src/app/pages/send/send-crypto-asset-form/form/stacks/use-stacks-common-send-form.tsx:31]: 😎

**Review by pete-watters — APPROVED** [review]
Great work @fbwoolf 🚀 

It's so satisfying to see:
Screenshot 2024-06-07 at 09 33 19

**Review by edgarkhanzadian — APPROVED** [review]
🚀 🚀 🚀

**Review by alter-eggo — DISMISSED** [review]
tested and lgtm

**Comment by alter-eggo** [comment]
@fbwoolf once submit stx tx I don't see pending tx in activity, though see this (1st screenshot)
seems in dev it's ok (2 screenshot)
Screenshot 2024-06-07 at 13 53 43
Screenshot 2024-06-07 at 13 56 01

**Review by kyranjamie — APPROVED** [review]

**Comment by fbwoolf** [comment]
> @fbwoolf once submit stx tx I don't see pending tx in activity, though see this (1st screenshot) seems in dev it's ok
>
Thx, will check this again, it was working ok but maybe something broke. 👀

**Pull request merged**